packages/contracts-bedrock/src/dispute/weth/WETH98.sol

```diff
@@ -36,6 +40,11 @@ contract WETH98 is IWETH {
         deposit();
     }
 
+    /// @notice Pipes to deposit.
+    fallback() external payable {
+        deposit();
+    }
+
```

Comment:
```suggestion
```

Reply:
We don't need a fallback here; `receive` will be triggered when positive `msg.value` is sent to the contract.

Reply:
Without the fallback defined explicitly, it impacts the ABI in a non backwards compat way. Is it the fact that fallback accepts `calldata.length > 0` where receive accepts `calldata.length == 0`?

Reply:
You can see this change in the e2e tests to handle the non backwards compat: https://github.com/ethereum-optimism/optimism/pull/10090

Reply:
> Is it the fact that fallback accepts calldata.length > 0 where receive accepts calldata.length == 0?

Yep, `receive` gets hit if `eq(calldatasize(), 0)` & `fallback` gets hit if `gt(calldatasize(), 0)` + no selectors match. But shouldn't this be the intended behavior? If I'm sending ETH to the `WETH98` contract, I send as a regular transfer (commonly `eq(calldatasize(), 0)`) or through a `payable` function. Is there a usecase for being able to send ETH through the fallback by providing useless calldata?

Reply:
The usecase is backwards compatibility with WETH9 as it supports this feature. Even though its seemingly useless, it preserves the ABI

Reply:
Okay, no worries. Let's keep it.